.github/workflows/linters.yml

```diff
@@ -0,0 +1,27 @@
+name: Linters
+
+on: pull_request
+
+env:
+  GITHUB_BASE_REF: ${{ github.base_ref }}
+
+jobs:
+  run-linters:
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Install phpcs
+        run: |
+          mkdir bin
+          curl -L https://squizlabs.github.io/PHP_CodeSniffer/phpcs.phar -o ./bin/phpcs
+          chmod +x ./bin/phpcs
+          git clone --depth 1 https://git.drupalcode.org/project/coder.git ./bin/drupal-coder
```

Comment:
Should we use civicrm coder here? I've been using civicrm coder for new projects because every now and then we keep updating the ruleset to exclude some drupal specific rules not relevant for civicrm. I intend switching those to use civicrm coder soon. 
But then this project looks a very small one so this might not really matter.